lib/utils.js

```diff
@@ -95,3 +95,62 @@ exports.findAvailablePort = function(app){
   });
 
 }
+
+exports.autoStoreData = function (req, res, next) {
```

Comment:
I think it should come before the function, and be a bit more descriptive - my preference would be that a user could get a full idea of the function and how to use it from the comment - not having to read the code.